draft-ietf-rtcweb-jsep.xml

```diff
@@ -623,10 +623,13 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             untouched if so. If the original resolution is too large for the
             attribute criteria, the sender SHOULD apply downscaling to the
             output of the MediaStreamTrack in order to satisfy the criteria.
-            In rare cases, where a receiver requires a minimum resolution which
-            is greater than the native resolution of the video, the
-            sender SHOULD apply upscaling in order to provide that resolution.
-            The sender SHOULD NOT apply upscaling in any other cases.
+            The sender MUST not apply upscaling if the upscaling policy as set
+            by the JavaScript forbids it. Otherwise, in rare cases, where a
+            receiver requires a minimum resolution which is greater than the
+            native resolution of the video, the sender SHOULD apply upscaling in
+            order to provide that resolution. The sender SHOULD NOT apply
+            upscaling in any other cases and MUST NOT if the JS policy does not
```

Comment:
This just seems duplicative. The idea is to have the logic:

If (!policy) {
  don't upscale
} else {
  if (minres > native_res) {
    SHOULD upscale
  } else {
    SHOULD NOT
  }
}